src/components/blipSelect/index.js

```diff
@@ -669,7 +686,7 @@ export class BlipSelect extends Component {
       selectOptionsContainer.style.opacity = 1
     })
 
-    this.input.parentNode.classList.add(bpInputWrapperFocusClass)
+    this.input.parentNode.parentNode.parentNode.classList.add(bpInputWrapperFocusClass)
```

Comment:
Maybe it's better to create a `getter` (like `get input()`) and return selected element to avoid to change this line whenever the input is repositioned.